.github/pull_request_template.md

```diff
@@ -1,7 +1,16 @@
-Fixes # .
+### Background 
+<!-- What was happening before this PR, and the problem(s) it solves -->
 
-Change summary:
-- 
+### Fixes 
```

Comment:
I'd move `Fixes` to the top of the template and will suffix it with the `#<issue number>` as shown below

### Fixes #<ISSUE_NUMER>

adding the issue number next to the `fixes` term will automatically link to the issue in the PR and will automatically close the issue once the PR has been merged.